### PR TITLE
add hyperv-csi-driver

### PR DESCRIPTION
**Pull request by Zetanova**
[hyperv-csi-driver](https://github.com/Zetanova/hyperv-csi-driver)

### CODE REVIEW

**Comment by Zetanova** [comment]
I signed it

File: book/src/drivers.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Hammerspace CSI](https://github.com/hammer-space/csi-plugin) | `com.hammerspace.csi` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Hammerspace Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot
 [Hitachi Vantara](https://knowledge.hitachivantara.com/Documents/Adapters_and_Drivers/Storage_Adapters_and_Drivers/Containers) | `com.hitachi.hspc.csi` | v1.0 | A Container Storage Interface (CSI) Driver for VSP series Storage | Persistent | ? | ? | 
 [HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[HyperV CSI](https://github.com/Zetanova/hyperv-csi-driver) | `eu.zetanova.csi.hyperv` | v0.1.0 | A Container Storage Interface (CSI) driver to manage hyperv hosts | Persistent | Read/Write Multiple Pods | Yes |
```
Comment (xing-yang) [inline book/src/drivers.md:41]: > v0.1.0

This is supported CSI spec version, not CSI driver version.  See other drivers for example.

**Comment by Zetanova** [comment]
I signed it
/check-cla

**Comment by Zetanova** [comment]
i messed up the rebase and recreating the PR

**Pull request closed without merging**